geo/Cargo.toml

```diff
@@ -13,10 +13,11 @@ rust-version = "1.75"
 categories = ["science::geo"]
 
 [features]
-default = ["earcutr", "spade"]
+default = ["earcutr", "spade", "i_overlay", "multithreading"]
 use-proj = ["proj"]
 proj-network = ["use-proj", "proj/network"]
 use-serde = ["serde", "geo-types/serde"]
+multithreading = ["i_overlay/allow_multithreading"]
```

Comment:
This `multithreading` feature also enables `i_overlay` - is that intended? 

If so, then I think we should call it something like `i_overlay_multithreading` (or per my other comment, `bool_ops_multithreading`)

Though I could imagine a use case for a `multithreading` flag in geo more generally. In which case this name is good as is! But we should change the definition to:

`multithreading = ["i_overlay?/allow_multithreading"]`

Which enables `allow_multithreading` on `i_overlay` if we're using `i_overlay`, but otherwise don't enable `i_overlay` just because someone tried to build geo with `multithreading`.

Reply:
My own preference is: 
`multithreading = ["i_overlay?/allow_multithreading"]`
(or `multithreading = ["bool_ops?/allow_multithreading"]`)

Reply:
I'll change it! I didn't know about this syntax